compiler/src/iree/compiler/Codegen/Common/Passes.h

```diff
@@ -149,6 +149,8 @@ struct GenericVectorizationPassOptions {
   bool enableCleanup = true;
   // Enable conversion for reduction ops to contraction ops.
   bool generateContract = true;
+  // Enable folding casting ops into contraction ops.
```

Comment:
Maybe expand the comment with something like:
```
// NOTE: the resulting mixed-type contraction ops are only handled by certain backends.
```
or some

Reply:
SGTM, thanks.